aleph/validation/schema/diagram.yml

```diff
@@ -0,0 +1,115 @@
+DiagramCreate:
+  type: object
+  required: ["data"]
+  properties:
+    data:
+      type: object
+      allOf:
+      - $ref: '#/components/schemas/DatedModel'
```

Comment:
I think that wants to move up one level.

Reply:
It's a type. Should be `DiagramData` instead.